compose/install/index.md

```diff
@@ -3,30 +3,73 @@ description: How to install Docker Compose
-* Docker Compose plugin requires Docker CLI.
+**To get Compose:**
 
-## Compose installation scenarios
-You can run Compose on macOS, Windows, and 64-bit Linux. Check what installation scenario fits your needs. 
+* If you have Docker Desktop, you've got a full Docker installation, including Compose.
 
-Are you looking to:
+* If you don't have Docker yet installed, or just need to install Compose, check the options below:
+{% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto; align=right"}' %}
 
-* __Get latest Docker Compose and its prerequisites__:
-[Install Docker Desktop for your platform](./compose-desktop.md). This is the fastest route and you get Docker Engine and Docker CLI with the Compose plugin. Docker Desktop is available for Mac, Windows and Linux.
+| Platform       | Docker Desktop                 | Compose CLI Plugin      | Compose Standalone  |
+|:---------------|:------------------------------:|:-----------------------:|:-------------------:|
+|Linux (64b)     |{{ yes }} [Install](../../desktop/install/linux-install.md)|{{ yes }} [Install](./compose-linux.md)|{{ yes }} [Install](./compose-linux.md#install-compose-standalone)|
+|Mac             |{{ yes }} [Install](../../desktop/install/mac-install.md)    | - | - |
+|Windows         |{{ yes }} [Install](../../desktop/install/windows-install.md)| - | - |
+|Windows Server  | - | - |{{ yes }} [Install](#install-compose-on-windows-server)|
 
-* __Install Compose plugin:__
-  + __(Mac, Win, Linux) Docker Desktop__: If you have Desktop installed then you already have the Compose plugin installed.
-  + __Linux systems__: To install the Docker CLI's Compose plugins use one of these methods of installation: 
-     + Using the [convenience scripts](../../engine/install/#server){: target="_blank" rel="noopener" class="_"} offered per Linux distro from the Engine install section.
-     + [Setting up Docker's repository](compose-plugin#install-using-the-repository) and using it to install the compose plugin package.     
-     + Other scenarios, check the [Linux install](compose-plugin#installing-compose-on-linux-systems).
-  + __Windows Server__: If you want to run the Docker daemon and client directly on Microsoft Windows Server, follow the [Windows Server install instructions](compose-plugin#install-compose-on-windows-server).
+**Table description:**
+* **Linux(64bit):** *option1)* Install [Docker Desktop for Linux](../../desktop/install/linux-install.md) or, *option2)* [Install Docker Engine and the CLI](../../engine/install/#server) and then [install Compose](compose-linux.md).
+* **macOS:** [Install Docker Desktop](../../desktop/install/mac-install/).
+* **Windows:** [Install Docker Desktop](../../desktop/install/windows-install/).
+* **Windows Server:** [Install Docker Engine and the CLI](../../engine/install/binaries/#install-server-and-client-binaries-on-windows) and then [install Compose](#install-compose-on-windows-server).
+
```

Comment:
This "table description" seems redundant with the information that's already in the table.  

Is this intended for accessibility, though? If so, that is unfortunately inconsistent with other instances of table usage. (e.g., on the DD [Linux Install](https://docs.docker.com/desktop/install/linux-install/#supported-platforms) page). 😞 

I would suggest that we remove from this section the repeated Docker Desktop download links, since it detracts from the simplicity and clarity of the links already in the table.

Without those links, it leaves the instructions for Linux and Windows Server. How about including them as clear sections?


### Note on Linux

```suggestion
### Note on Linux

On Linux(64bit), you may want to install Docker Engine separately from Compose, instead of the approach of installing everything with Docker Desktop. [This gives you more control over your install location and version of Compose.]

 1. [Install Docker Engine and the CLI](../../engine/install/#server) 
 2. [install Compose](compose-linux.md).


Alternately, you may want to [install Compose standalone], without the pre-requisite of installing the Docker Engine. 


###  Note on Windows Server
(This then replaces "Install Compose on Windows Server")
```


> [This gives you more control over your install location and version of Compose.]

I just put this here as a placeholder for trying to explain a common use case for bypassing the Docker Desktop install. 

Is there more we can say here to address "Under what circumstances would the user need to install Compose outside Docker Desktop? And how do they tell the difference between whether they should install it as a plugin, or standalone?"

Reply:
See closing comment.